Tests/DatadogTests/Datadog/RUMMonitorTests.swift

```diff
@@ -540,14 +540,17 @@ class RUMMonitorTests: XCTestCase {
         let firstViewEvent = try rumEventMatchers
             .lastRUMEvent(ofType: RUMDataView.self) { rumModel in rumModel.view.url == "FirstViewController" }
 
-        XCTAssertEqual(try firstViewEvent.attribute(forKeyPath: "attribute1") as String, "changed value 1")
-        XCTAssertEqual(try firstViewEvent.attribute(forKeyPath: "attribute2") as String, "value 2")
+        XCTAssertNil(try? firstViewEvent.attribute(forKeyPath: "attribute1") as String)
+        XCTAssertNil(try? firstViewEvent.attribute(forKeyPath: "attribute2") as String)
+        XCTAssertEqual(try firstViewEvent.attribute(forKeyPath: "context.attribute1") as String, "changed value 1")
+        XCTAssertEqual(try firstViewEvent.attribute(forKeyPath: "context.attribute2") as String, "value 2")
 
         let secondViewEvent = try rumEventMatchers
             .lastRUMEvent(ofType: RUMDataView.self) { rumModel in rumModel.view.url == "SecondViewController" }
 
-        XCTAssertEqual(try secondViewEvent.attribute(forKeyPath: "attribute1") as String, "changed value 1")
+        XCTAssertNil(try? secondViewEvent.attribute(forKeyPath: "attribute1") as String)
         XCTAssertNil(try? secondViewEvent.attribute(forKeyPath: "attribute2") as String)
```

Comment:
Here we should test that second attribute is not encoded, as it was removed:
```suggestion
        XCTAssertNil(try? secondViewEvent.attribute(forKeyPath: "context.attribute2") as String)
```